client/client-multi/client-c/src/kaa/kaa_logging.c

```diff
@@ -263,15 +282,17 @@ static void update_storage(kaa_log_collector_t *self)
 
 
 
-kaa_error_t kaa_logging_add_record(kaa_log_collector_t *self, kaa_user_log_record_t *entry)
+kaa_error_t kaa_logging_add_record(kaa_log_collector_t *self, kaa_user_log_record_t *entry, kaa_log_record_info_t *log_info)
 {
     KAA_RETURN_IF_NIL2(self, entry, KAA_ERR_BADPARAM);
     KAA_RETURN_IF_NIL(self->log_storage_context, KAA_ERR_NOT_INITIALIZED);
 
     kaa_log_record_t record = { NULL, entry->get_size(entry) };
     if (!record.size) {
-        KAA_LOG_ERROR(self->logger, KAA_ERR_BADDATA, "Failed to add log record: serialized record size is null."
-                                                                                "Maybe log record schema is empty");
+        KAA_LOG_ERROR(self->logger,
+                      KAA_ERR_BADDATA, "Failed to add log record: "
+                                       "serialized record size is null. "
+                                       "Maybe log record schema is empty");
```

Comment:
It's better to either move `KAA_ERR_BADDATA` to first line, or move string to a separate line.